Doctrine/Phpcr/MediaManager.php

```diff
@@ -133,8 +134,12 @@ public function setDefaults(MediaInterface $media, $parentPath = null)
      */
     public function mapPathToId($path, $rootPath = null)
     {
-        // The path is being the id
-        $id = PathHelper::absolutizePath($path, '/');
+        try {
+            // The path is being the id
+            $id = PathHelper::absolutizePath($path, '/');
+        } catch (RepositoryException $e) {
+            throw new \OutOfBoundsException($e->getMessage(), 0, $e);
```

Comment:
we could use Symfony\Component\Routing\Exception\ResourceNotFoundException i think.

can you please create the exception like this: `('Not a valid PHPCR path', 0, $e)` - the exception from PHPCR is probably not explaining the context.